actions/workflows/st2workroom_test.yaml

```diff
@@ -76,7 +76,29 @@
         distro: "{{distro}}"
         user: "{{st2_username}}"
         pass: "{{st2_password}}"
+      on-success: "verify_no_default_private_ssh_key_is_installed_for_stanley_1"
```

Comment:
Could these test be a separate workflow simply called from here? I suspect these will grow.